tools/functional-tester/etcd-runner/main.go

```diff
@@ -212,6 +219,136 @@ func runRacer(eps []string, round int) {
 	doRounds(rcs, round)
 }
 
+func runWatcher(eps []string) {
+	runningTime := 60 * time.Second // time for which operation should be performed
+	noOfPrefixes := 36              // total number of prefixes which will be watched upon
+	watchPerPrefix := 10            // number of watchers per prefix
+	reqRate := 20                   // put request per second
```

Comment:
@xiang90 just a thought if we make the request to 200 and with running time as 60, the it will be able to put 12000 keys and keeping prePrefixKey to 100 and prefixes as 36 will only be able to have a total of 3600 keys which will be quite less, i guess. So may be we can have prePrefix keys as 200 so that total no of keys to be put comes to 7200, which will enable it to finish well before time and also have a good number of keys to be put.
What do you suggest?